src/core/components/AMInstallButton/index.js

```diff
@@ -130,7 +140,24 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
   };
 
   installExtension = async (event: SyntheticEvent<HTMLAnchorElement>) => {
-    const { addon, enable, install, isAddonEnabled } = this.props;
+    const {
+      _tracking,
+      addon,
+      enable,
+      install,
+      isAddonEnabled,
+      variant,
+    } = this.props;
+
+    const category = `${EXPERIMENT_CATEGORY_CLICK}-${
+      !addon.is_recommended ? 'not_' : ''
+    }recommended`;
+    _tracking.sendEvent({
+      // If the experiment is not enabled we still want to send the event.
+      action: variant || NOT_IN_EXPERIMENT,
```

Comment:
Why is this variable/value called `NOT_IN_EXPERIMENT`? Shouldn't this be `EXPERIMENT_DISABLED` (`experimentDisabled`)? That would avoid the comment above, no?